srv/modules/runners/populate.py

```diff
@@ -1077,9 +1059,9 @@ def engulf_existing_cluster(**kwargs):
 
     search = __utils__['deepsea_minions.show']()
     local = salt.client.LocalClient()
-    settings = Settings()
-    salt_writer = SaltWriter(overwrite=True)
     exception = kwargs.get('exception', False)
+    settings = __utils__['settings.self_']()
+    salt_writer = SaltWriter(**kwargs)
```

Comment:
Should be `salt_writer = SaltWriter(overwrite=True)` (see 51a06fa9)

Reply:
ack